ubg/src/mafia/MafiaGame.js

```diff
@@ -33,6 +34,9 @@ function MafiaGame({day, room, usersCollection, usersData, userUid,
   const classes = useStyles();
   const [alert, setAlert] = useState(null);
   const [userInfo, setUserInfo] = useState('');
+  const userDoc = useFirestore().collection('users').doc(userUid);
+  const userDocData = useFirestoreDocData(userDoc);
```

Comment:
Maybe use the redux state here, we have the array of all users in room maybe just filter that to get the current user?